### PR TITLE
Origin/810 historical index cases gisida lite progressing

### PR DESCRIPTION
**Pull request by p-netm**

### CODE REVIEW

File: src/containers/pages/FocusInvestigation/map/active/helpers/utils.tsx [hunk 1]
```diff
@@ -189,3 +212,174 @@ export const getDetailViewPlanInvestigationContainer = (plan: Plan): React.React
...
+/** build layers for all other points , polygons and multi-polygons
+ * @param {string | null} currentGoal - goal id for the layer that is being built
+ * @param {FeatureCollection<TaskGeoJSON>} pointFeatureCollection- feature collection for points
+ * @param {FeatureCollection<TaskGeoJSON>} polygonFeatureCollection - feature collection for polygons and multi-polygons
+ * @param {ExtraVars} - some vars to help in if branches within the function
+ */
+export const buildGsLiteLayers = (
+  currentGoal: string | null,
+  pointFeatureCollection: any,
+  polygonFeatureCollection: any,
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/map/active/helpers/utils.tsx:287]: Do these have to be of type any?

File: src/containers/pages/FocusInvestigation/map/active/tests/index.test.tsx [hunk 2]
```diff
@@ -601,41 +640,10 @@ describe('containers/pages/FocusInvestigation/activeMap', () => {
       </Provider>
     );
     await new Promise<unknown>(resolve => setImmediate(resolve));
-    expect(displayErrorMock.mock.calls.length).toBe(1);
+    expect(displayErrorMock).toHaveBeenCalled();
```
Comment (moshthepitt) [inline src/containers/pages/FocusInvestigation/map/active/tests/index.test.tsx:643]: Can we ascertain, in this test, how many times this was called?

File: src/containers/pages/FocusInvestigation/map/active/helpers/utils.tsx (hunk 1 above)
Comment (p-netm) [reply src/containers/pages/FocusInvestigation/map/active/helpers/utils.tsx:287]: nope

File: src/containers/pages/FocusInvestigation/map/active/tests/index.test.tsx (hunk 2 above)
Comment (p-netm) [reply src/containers/pages/FocusInvestigation/map/active/tests/index.test.tsx:643]: ok.

**Pull request merged**